### ISSUE TITLE
Rule proposal: Require alt attribute for image tag

### ISSUE THREAD

**Issue by manniL** [vuejs/eslint-plugin-vue#466]
**Please describe what the rule should do:**

Ensure there is a non-empty alt attribute when using the `<img>` tag.

**What category of rule is this? (place an "X" next to just one item)**

[X] Enforces code style
[ ] Warns about a potential error
[ ] Suggests an alternate way of doing something
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```html
<img src="path/to/my/src" alt="">

<img src="path/to/my/src">
```

**Comment by michalsnik** [comment]
Thanks for posting this proposal @manniL But there is already a dedicated plugin with accessibility rules, that has this rule: https://github.com/maranran/eslint-plugin-vue-a11y/blob/master/docs/rules/alt-text.md
